airbyte-scheduler/src/main/java/io/airbyte/scheduler/temporal/CheckConnectionWorkflow.java

```diff
@@ -97,13 +97,12 @@ public JobOutput run(IntegrationLauncherConfig launcherConfig, StandardCheckConn
         final IntegrationLauncher integrationLauncher =
             new AirbyteIntegrationLauncher(launcherConfig.getJobId(), launcherConfig.getAttemptId().intValue(), launcherConfig.getDockerImage(), pbf);
         final AirbyteStreamFactory streamFactory = new DefaultAirbyteStreamFactory();
-        final OutputAndStatus<JobOutput> run =
-            new JobOutputCheckConnectionWorker(new DefaultCheckConnectionWorker(integrationLauncher, streamFactory)).run(connectionConfiguration,
-                jobRoot);
+        final OutputAndStatus<StandardCheckConnectionOutput> run =
+            new DefaultCheckConnectionWorker(integrationLauncher, streamFactory).run(connectionConfiguration, jobRoot);
         if (run.getStatus() == JobStatus.SUCCEEDED) {
           Preconditions.checkState(run.getOutput().isPresent());
           LOGGER.info("job output {}", run.getOutput().get());
-          return run.getOutput().get();
+          return new JobOutput().withCheckConnection(run.getOutput().get());
```

Comment:
each workflow does this rewrapping. this is going away in the next PR when i change the interface on these workflows, so please tolerate it for one more PR!